templates/display.tpl

```diff
@@ -12,7 +12,13 @@
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset={$defaultCharset|escape}" />
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>{translate key="article.pageTitle" title=$title|escape}</title>
+	<title>
+	{if $isTitleHtml}
+		{translate key="article.pageTitle" title=$title|escape}
+	{else}
+		{translate key="article.pageTitle" title=$title|strip_tags|escape}
+	{/if}
```

Comment:
The condition/code needs to be inverted.

Reply:
@jonasraoni Thanks for pointing this out! I've updated the code and inverted it.